src/codegen.jl

```diff
@@ -497,6 +497,27 @@ function xcall(m::Module, name::Symbol, args...; kw...)
     xcall(Expr(:., m, QuoteNode(name)), args...; kw...)
 end
 
+"""
+    xblock(x)
+
+Create a block expression around `x`.
+"""
+xblock(x) = Expr(:block, x)
+
+"""
+    xsplat(x)
+
+Create a splatting expression around `x`, i.e. `x...`.
+"""
+xsplat(x) = Expr(:(...), x)
+
+"""
+    xannot(x, t)
+
+Create a type annotation expression around `x` for type `t`, i.e. `x::t`.
+"""
+xannot(x, y) = Expr(:(::), x, y)
```

Comment:
I'm a bit hesitate to have this btw, I think `:($x::$y)` is already simple enough for creating annotations and it is probably more readable, why do we need a function for it?